### PR TITLE
fix: only pages from /index.md/ #34

### PR DESCRIPTION
**Pull request by olavea**

### CODE REVIEW

File: gatsby-node.js [hunk 1]
```diff
@@ -13,68 +16,58 @@ async function slugifyMarkdownRemarkNode({ actions, node, getNode }) {
...
+  // Loop over the supplies.nodes and
+  // for each aromaNode bake a page
   data.supplies.nodes.forEach((aromaNode) => {
-    console.log(aromaNode.fields.slug, "💀📄");
-
-    const aromaNodePath =
-      aromaNode.fields.slug === "/index/" ? "/" : aromaNode.fields.slug;
+    // console.log(aromaNode.fields.slug, "💀📄");
+    const aromaNodeSlug = aromaNode.fields.slug;
+    const aromaNodePath = aromaNodeSlug === "/index/" ? "/" : aromaNodeSlug;
```
Comment (raae) [inline gatsby-node.js:50]: I like to refactor so it stays one one line when possible, so that is what I did here.

File: gatsby-node.js [hunk 2]
```diff
@@ -13,68 +16,58 @@ async function slugifyMarkdownRemarkNode({ actions, node, getNode }) {
...
       // B. bakingSong 🎵 🙀
       component: bakingSong,
-
       // C. catsbyId 😼🆔
       context: {
         catsbyId: aromaNode.id,
       },
     });
+
+    reporter.info(`Created page for slug ${aromaNode.fields.slug}`);
```
Comment (raae) [inline gatsby-node.js:63]: Added a reporter so its easier to follow what is happening

**Review by raae — APPROVED** [review]

File: gatsby-node.js (hunk 1 above)
Comment (olavea) [reply gatsby-node.js:50]: 🚀

File: gatsby-node.js (hunk 2 above)
Comment (olavea) [reply gatsby-node.js:63]: 🚀

**Pull request merged**